.travis.yml

```diff
@@ -24,9 +26,12 @@ before_install:
 - export PATH=/home/travis/miniconda/bin:$PATH;
 
 install:
-- conda install --yes python=$PY numpy scipy nose sphinx mock swig pip matplotlib numpydoc;
+- conda install --yes python=$PY nose sphinx mock swig pip numpydoc;
 - pip install --upgrade pip;
-- pip install mpi4py;
+- pip install mpi4py
```

Comment:
I'm not convinced why we need `mpi4py` since AFAIK we are not using any parallel capability of OpenMDAO. Not sure why there was an error during docs build, that looks like an internal OpenMDAO error to me.